projects/sandbox/train/train/utils.py

```diff
@@ -53,110 +35,137 @@ def split(X: Tensor, frac: float, axis: int) -> Tuple[Tensor, Tensor]:
         return torch.split(X, splits, dim=axis)
 
 
-def prepare_augmentation(
-    glitch_dataset: Path,
-    waveform_dataset: Path,
-    ifos: List[str],
-    train_start: float,
-    train_stop: float,
-    glitch_prob: float,
-    waveform_prob: float,
-    glitch_downweight: float,
-    swap_frac: float,
-    mute_frac: float,
-    sample_rate: float,
-    highpass: float,
-    mean_snr: float,
-    std_snr: float,
-    min_snr: float,
-    trigger_distance: float = 0,
-    valid_frac: Optional[float] = None,
-):
-    # build a glitch sampler from a pre-saved bank of
-    # glitches which will randomly insert them into
-    # either or both interferometer channels
-    glitch_dict = {}
-    valid_glitches_list = []
+def get_background(background_dir: Path):
+    # load in the data from the first segment in the background directory
+    try:
+        background_dataset = next(background_dir.iterdir())
```

Comment:
Can we confirm that these come sorted?

Reply:
I don't think they do but since we only download one training background segment for now it shouldn't be a problem. But this should definitely be made more general when we overhaul the data picture.